vaadin-grid-pro-flow-integration-tests/src/test/java/com/vaadin/flow/component/gridpro/test/BasicIT.java

```diff
@@ -26,6 +27,12 @@ public void editColumnsAdded() {
         Assert.assertEquals(columns.size(), 3);
     }
 
+    @Test
+    public void columnIsRenderedInBeanGrid() {
+        GridTHTDElement cell = beanGrid.getCell(0, 0);
+        Assert.assertEquals("Person 1", cell.getInnerHTML());
```

Comment:
Also verify that the editor can be opened to make sure that the overload actually creates an edit-column instead of a normal one.